components/VideoPlayer.js

```
@@ -109,10 +109,14 @@
 						setIsPresenting(true);
 						rootStore.set({ isFullscreen: true });
 						break;
-					case VideoFullscreenUpdate.PLAYER_DID_PRESENT:
-						setIsPresenting(false);
-						break;
-					case VideoFullscreenUpdate.PLAYER_WILL_DISMISS:
+                                        case VideoFullscreenUpdate.PLAYER_DID_PRESENT:
```

ESLint

[error] Expected indentation of 5 tabs but found 40 spaces.

(indent)